src/ml/neural_net/mps_layer_instance_norm_data_loader.mm

```diff
@@ -159,26 +157,47 @@ - (void) setLearningRate:(float)lr {
   [_adamBeta setLearningRate:lr];
 }
 
-- (void) updateCurrentStyle:(NSUInteger)style {
+- (void) updateIndex:(NSUInteger)style {
   _currentStyle = style;
 }
 
 - (void) loadBeta:(float *)beta {
-  memcpy(_beta_weights.mutableBytes, beta, _numberOfFeatureChannels * _styles * sizeof(float));
+  float* betaWeights = (float *) [[[[style_props objectAtIndex: _currentStyle] betaVector] data] contents];
+  memcpy(betaWeights, beta, _numberOfFeatureChannels * _styles * sizeof(float));
 }
 
 - (float *) beta {
-  [self checkpointWithCommandQueue:_cq];
-  return (float *) [[[[style_props objectAtIndex: _currentStyle] betaVector] data] contents];
+  NSUInteger previousStyle = _currentStyle;
+  NSMutableData * betaPlaceHolder = [NSMutableData data];
+  for (NSUInteger index = 0; index < _styles; index++) {
+    _currentStyle = index;
+    [self checkpointWithCommandQueue:_cq];
+    float* betaWeights = (float *) [[[[style_props objectAtIndex: _currentStyle] betaVector] data] contents];
+    [betaPlaceHolder appendBytes:betaWeights length:sizeof(float)*_numberOfFeatureChannels];
+  }
+  _currentStyle = previousStyle;
+
+  return (float *) (betaPlaceHolder.bytes);
 }
 
 - (void) loadGamma:(float *)gamma {
-  memcpy(_gamma_weights.mutableBytes, gamma, _numberOfFeatureChannels * _styles * sizeof(float));
+  float* gammaWeights = (float*) [[[[style_props objectAtIndex: _currentStyle] gammaVector] data] contents];
```

Comment:
`style_props` should have an underscore so that the reader knows it's an instance variable

Reply:
Alright! Added underscore!